docs/fast_data/configuration/realtime_updater/common.md

```diff
@@ -308,25 +309,10 @@ The message should include the following properties:
 - `value`: it is **null** if it's a *delete* operation, otherwise it contains the data of the projection.
 - `offset`: it is the offset of the Kafka message.
 
-Example of a delete operation
-
-```text
-key: `{"USER_ID": 123, "FISCAL_CODE": "ABCDEF12B02M100O"}`
-value: null
-timestamp: '1234556789'
-offset: '100'
-```
-
-Example of an upsert:
-```text
-key: `{"USER_ID": 123, "FISCAL_CODE": "ABCDEF12B02M100O"}`
-value: `{"NAME": 456}`
-timestamp: '1234556789'
-offset: '100'
-```
-
 These are the only fields needed to configure correctly the message adapter. For more details and further explanations, you can read the [documentation page about the supported JSON format](https://www.ibm.com/docs/en/idr/11.4.0?topic=kcop-write-json-format-records).
 
+To see the message's structure specification and some examples go to the [Inputs and Outputs page](/fast_data/inputs_and_outputs.md#ibm-infosphere-data-replication-for-db2) .
```

Comment:
are you sure this link is correct? It has the anchor to db2

Reply:
Yeah it takes you to the inputs and outputs page to the async api specification of the DB2 messages. Just checked the link and it works.